### PR TITLE
Fix broken Gui

### PR DESCRIPTION
**Pull request by nemaniarjun**
Fix mistakes made in previous commits.
Renamed functions are now called correctly.
After a `Flush` the started download will update
the UI.

### CODE REVIEW

**Comment by abhsag24** [comment]
Hi thank you for your Patch make sure you have the right commit structure that is 
```
tag: Shortlog

body
Fixes <issue number/url>
```

File: uiplib/gui/gallery.py [hunk 1]
```diff
@@ -73,7 +74,7 @@ def update(self):
 
         if len(appObj.images) != 0:
             self.set_image(
-                appObj.images[(appObj.index) % len(appObj.images)])
+                choice(appObj.images))
```
Comment (nkprince007) [inline uiplib/gui/gallery.py]: could you make it a single line?

Comment (nemaniarjun) [inline uiplib/gui/gallery.py]: Okay

File: uiplib/gui/mainGui.py [hunk 2]
```diff
@@ -84,6 +84,7 @@ def flush(self):
                                 args=(self.settings['website'],
                                       self.settings['pics-folder'],
                                       self.settings['no-of-images']),
+                                kwargs={'appObj': self},
```
Comment (nkprince007) [inline uiplib/gui/mainGui.py]: Nice 😉

File: uiplib/scheduler.py [hunk 3]
```diff
@@ -108,5 +108,3 @@ def run(self):
             self.change_random()
             self.setStartTime(time.time())
             self.changeCycle()
-        else:
```
Comment (nkprince007) [inline uiplib/scheduler.py]: why was this removed?

Comment (nemaniarjun) [inline uiplib/scheduler.py]: Because the download is threaded, when this line is run, the folder is empty. Which prints this false message
